### PR TITLE
Add Comment Marks package

### PR DESCRIPTION
**Pull request by maegul**

### CODE REVIEW

**Comment by maegul** [comment]
Not trying to nag here ... but have I done something wrong with this submission/pull-request?

**Comment by braver** [comment]
Please resolve the conflict and maybe have a look at our [new checklist](https://github.com/wbond/package_control_channel/blob/master/.github/PULL_REQUEST_TEMPLATE.md)

**Comment by braver** [comment]
Please respond to the feedback to continue this PR.

**Comment by maegul** [comment]
New Checklist:

- [X] I'm the package's author and/or maintainer.
- [X] I have have read [the docs][1].
- [X] I have tagged a release with a [semver][2] version number.
- [X] My package repo has a description and a README describing what it's for and how to use it.
- [X] My package doesn't add context menu entries. *
- [X] My package doesn't add key bindings. **
- [X] Any commands are available via the command palette.
- [X] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
	- Keybindings only available through menu (which is detailed in the README)
- [X] If my package is a syntax it doesn't also add a color scheme. ***
- [ ] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

File: repository/c.json [hunk 1]
```diff
@@ -3003,6 +3003,18 @@
...
+			"name": "Comment Marks",
+			"details": "https://github.com/maegul/comment_marks",
+			"labels": ["comments", "code navigation"],
+			"author": "maegul",
+			"releases": [
+				{
+					"sublime_text": ">=4050",
+          "tags": true
+        }
+      ]
```
Comment (braver) [inline repository/c.json:3015]: indentation got kinda messy here

**Comment by braver** [comment]
It really helps if you tell us a little bit about your package. In any case you should add a little description to your GitHub repo, that's picked up by package control and helps people find your package and understand what it's for. 
It also looks like your key bindings would also get loaded (ST doesn't really care about the filename and will probably load all `*.sublime-keymap` files it can find.

**Comment by maegul** [comment]
> It really helps if you tell us a little bit about your package. In any case you should add a little description to your GitHub repo, that's picked up by package control and helps people find your package and understand what it's for.

* What's wrong with the already existing [readme](https://github.com/maegul/comment_marks/blob/0.1.9/README.md)?

> It also looks like your key bindings would also get loaded (ST doesn't really care about the filename and will probably load all `*.sublime-keymap` files it can find.

* This isn't the case on my machine (FWIW).  
* Also the `Brackethighlighter` package does something similar (see [Menu File here](https://github.com/facelessuser/BracketHighlighter/blob/223ffd4ceafd58686503e3328934c039e959a88c/Main.sublime-menu#L252)), which appears to work as intended (ie, the `Example.sublime-keymap` file is not loaded) on my machine.
* [Unofficial Docs](https://docs.sublimetext.io/reference/key_bindings.html#naming-keymap-files) indicate that keymap files must start with `"Default"`.
* [Official Docs](https://www.sublimetext.com/docs/key_bindings.html#user_bindings) on User defined key binding files, are consistent with this, though not conclusive.

Some clarity around this might be good, especially if it's going to change.  Otherwise, do you have any suggestions on how to do something similar?  What I like about this approach is that it's low maintenance for package authors but still convenient for users given the item in the Preferences Menu.

**Comment by braver** [comment]
> Unofficial Docs indicate that keymap files must start with "Default".

I learned something new 😄 I didn’t know it was specific about the file names for keymaps. I don’t have any other information than you have access to, so I would expect that if it works as intended now it should stay that way (at least for this major version of ST). So all good there 👍🏻

> What's wrong with the already existing readme?

The description is used for these lines:

Braver - Search - Package Control

**Comment by maegul** [comment]
> I learned something new 😄 I didn’t know it was specific about the file names for keymaps. I don’t have any other information than you have access to, so I would expect that if it works as intended now it should stay that way (at least for this major version of ST). So all good there 👍🏻

Cheers.  Though I do feel like I (and any other package author relying on this) ought to be more certain than I am ... might duck into the discord on this at some point.

> The description is used for these lines:

Thanks!  Sorry for my daftness ... I had forgotten about the GitHub descriptions.  Done now ... [repo here](https://github.com/maegul/comment_marks)

**Comment by jrappen** [comment]
Not sure why this PR used mixed indentation.

**Comment by jrappen** [comment]
Also wrong order in this PR was fixed in wbond/package_control_channel@f63fd2b.

**Pull request merged**